### ISSUE TITLE
Version 2.4.0 missing in the download archive

### ISSUE THREAD

**Issue by ccztux** [NagiosEnterprises/ncpa#931]
Version `2.4.0` is missing in the [download archive](https://www.nagios.org/ncpa/archive.php):

![image](https://user-images.githubusercontent.com/11867479/226554292-84445948-3ec7-4ea1-b069-e7d98e021a3f.png)

**Comment by corynorell** [comment]
You're right, thank you for bringing this to our attention. I'll look into it shortly.
